docs/command-reference/server-management/client-tracking.md

```diff
@@ -0,0 +1,70 @@
+---
+description:  Learn how to use Redis CLIENT TRACKING command to control server-assisted client side caching for the connection.
+---
+
+import PageTitle from '@site/src/components/PageTitle';
+
+# CLIENT TRACKING
+
+<PageTitle title="Redis CLIENT TRACKING Command (Documentation) | Dragonfly" />
+
+## Syntax
+
+    CLIENT TRACKING <ON | OFF>
+
+**Time complexity:** O(1). Some options may introduce additional complexity.
+
+**ACL categories:** @slow, @connection
```

Comment:
@romange @kostasrim i just have a question, do you know why Redis mark this as slow? I'm don't really understand this tag... itself is a fast command...